test/views/behaviour.js

```diff
@@ -0,0 +1,193 @@
+
+/**
+ * Behaviour of the Echidna testbed.
+ *
+ * @author Antonio <antonio@w3.org>
+ */
+
+'use strict';
+
+var JOBS = ['retrieve-resources', 'specberus', 'token-checker', 'third-party-checker', 'publish', 'tr-install', 'update-tr-shortlink'];
+
+var log;
+var inspector;
+var allSpecs = [];
+var allJobs = {};
+var allMessages = {};
+
+function logMessage(message) {
+
+    log = log || $('pre#console');
+    log.append(new Date().toLocaleTimeString() + '  ' + message + '\n');
+
+}
+
+function dumpObject(obj) {
+
+    inspector = inspector || $('pre#inspector');
+
+    if (!obj) {
+        inspector.text('');
+    }
+    else {
+        inspector.text(JSON.stringify(obj, null, 2));
+    }
```

Comment:
Again, our preference about this isn't decided yet. Douglas Crockford and others advocate always wrapping blocks in curly brackets, to avoid common mistakes, and I think it's a good idea.

(Apart from that, having the condition and the statement like you suggest, together in the same line, is much harder to read, don't you think?)